websites/C/Cuevana 3/presence.ts

```diff
@@ -30,47 +33,112 @@ presence.on(
+					(presenceData.smallImageText = videoPaused
+						? strings.pause
+						: strings.play),
+					(presenceData.startTimestamp = startTimestamp),
+					(presenceData.endTimestamp = endTimestamp);
+
+				if (videoPaused) {
+					delete presenceData.startTimestamp;
+					delete presenceData.endTimestamp;
+				}
+			}
+
+			presence.setActivity(presenceData, !videoPaused);
+		} else if (document.location.pathname.includes("/serie/")) {
+			presenceData.details = document.querySelector("h1.Title").textContent;
+			presenceData.smallImageKey = "browsing";
+			presenceData.smallImageText = strings.browsing;
+			presenceData.largeImageKey = document
+				.querySelector("div.backdrop > article > div.Image > figure > img")
+				.getAttribute("data-src");
+		} else {
+			presenceData.details = strings.browsing;
+			presenceData.smallImageKey = "browsing";
+			presenceData.smallImageText = strings.browsing;
+		}
+	}
+
 	const presenceData: PresenceData = {
 		largeImageKey: "logo",
 	};
 
 	if (
-		document.location.pathname.match("^/[0-9]") ||
+		document.location.pathname.includes("/pelicula") ||
 		document.location.pathname.includes("/episodio")
 	) {
-		const titulo = document.querySelector("h1.Title").textContent,
-			subtitulo = document.querySelector("h2.SubTitle").textContent,
-			cover = document
-				.querySelector("div.backdrop > article > div.Image > figure > img")
-				.getAttribute("data-src");
+		let titulo, cover;
 
 		if (!document.location.pathname.includes("/episodio")) {
+			titulo = document.querySelector("h1.title > span").textContent;
 			presenceData.details = titulo;
-			if (subtitulo === titulo) delete presenceData.state;
-			else presenceData.state = subtitulo;
+			delete presenceData.state;
 			presenceData.buttons = [
 				{ label: "Ver Película", url: window.location.href },
 			];
+			cover = `https://ww3.cuevana.pro${String(
+				document.querySelector("figure.poster > img").getAttribute("src")
+			)}`;
 			presenceData.largeImageKey = cover;
 		} else {
-			presenceData.details = titulo
-				.replace(document.querySelector("h1.Title > span").textContent, "")
-				.replace(/\s+/g, " ")
-				.trim();
-			presenceData.state = `${
-				document.querySelector("h1.Title > span").textContent
-			} ${subtitulo}`;
+			titulo = document.querySelector("h1.title").textContent;
+			presenceData.details = titulo;
+			delete presenceData.state;
 			presenceData.buttons = [
 				{ label: "Ver Episodio", url: window.location.href },
 			];
-			presenceData.largeImageKey = cover;
+			presenceData.largeImageKey = document
+				.querySelector("figure > img")
+				.getAttribute("src");
 		}
 
 		if (iFrameVideo) {
 			(presenceData.smallImageKey = videoPaused ? "pause" : "play"),
 				(presenceData.smallImageText = videoPaused
-					? (await strings).pause
-					: (await strings).play),
+					? strings.pause
+					: strings.play),
 				(presenceData.startTimestamp = startTimestamp),
 				(presenceData.endTimestamp = endTimestamp);
```

Comment:
The extra brackets are not needed

Reply:
🤔

Reply:
how am i this bad

Reply:
oh i think it messed up when i clicked add commit on github

Reply:
OH @EncryptedDev yarn format likes it like that